install/kubernetes/helm/istio/values-istio-demo-common.yaml

```diff
@@ -0,0 +1,59 @@
+# This is used to generate minimal demo mode. It is included from demo and demo-auth values.
+# It is shipped with the release, used for bookinfo or quick installation of istio.
+# Includes components used in the demo, defaults to alpha3 rules.
+global:
+  proxy:
+    accessLogFile: "/dev/stdout"
+    resources:
+      requests:
+        cpu: 10m
+        memory: 40Mi
+
+  disablePolicyChecks: false
+  outboundTrafficPolicy:
+    mode: REGISTRY_ONLY
```

Comment:
why is this set to registry_only?

Reply:
Did we change the default or for demo?
I can change this back.
 I did comparison of istio-demo.yaml before and aft3r, and it came back ok.

Reply:
we changed for both (default + demo).